### DISCUSSION TITLE
Git clone error : Crazyflie firmware

### DISCUSSION THREAD

**Discussion by NageshEranki** [bitcraze/discussions#1318] [Q&A]
Hi 

I was trying to start from scratch and re-download the CF firmware. I followed the usual git commands as described under 'build and flash instructions'. However, I see the following error that I can't explain:

```Error downloading object: CMSIS/DSP/Lib/ARM/arm_ARMv8MBLl_math.lib (496f8cd): Smudge error: Error downloading CMSIS/DSP/Lib/ARM/arm_ARMv8MBLl_math.lib (496f8cdab57174e66eb6cfb365cba46298b86440ddaf1660176bcac0566aebb6): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.```

Current environment:

macOS Sonoma 14.4.1
Apple M3 chip
Running Rosetta terminal

**Comment by ataffanel** [comment]
This seems to be a transient error since it is working now for me so I expect it to work for you as well if you try again.

This is a quite worrying problem though since we have no control on this repository, this is a dependency to us. We will investigate what could have happened and if we need to do anything about it. Thanks for reporting the problem!

**Comment by NageshEranki** [comment]
Hi 

I tried again: once after brew reinstalling the arm toolchain, restarting the laptop and once after switching to my native arm terminal. No luck. I have attached the error logs with this post in case it would help us find out more about the issue:

<img width="1268" alt="Screenshot 2024-05-02 at 9 01 42 AM" src="https://github.com/bitcraze/discussions/assets/28665953/62bafb37-0c35-495f-994c-093ab24580d2">


Edit: Attached the full error logs saved to file

[20240502T085951.391025.log](https://github.com/bitcraze/discussions/files/15189539/20240502T085951.391025.log)

**Reply by gemenerik** [reply]
We're investigating a fix
